### PR TITLE
new icon: detaspace (original, original-wordmark, line, line-wordmark)

### LINKED ISSUE #1845: [ICON REQUEST]: Deta Space
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Deta Space is a serverless cloud hosting platform focused on ease of development and quick deployment/scaling. Even though this is a hosting platform I feel it should be added since Vercel has been added.

### Links and sources

https://deta.space
https://github.com/deta
https://deta.space/landing-page/assets/logo.20539aa2.svg
https://github.com/deta/space-docs/blob/main/.github/deta.svg

### Additional information

I can create the PR if the icon is accepted.

### PR DESCRIPTION
**Pull request by lemonyte**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/wiki/Overview-on-Submitting-Icons)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/wiki/Organizing-SVGs)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [x] A new object is added in the `devicon.json` file at the correct alphabetic position as seen [here](https://github.com/devicons/devicon/wiki/Updating-%60devicon.json%60)

- **This PR closes #1845**

## Link to prove your SVG is correct and up-to-date.

https://deta.space/
https://github.com/deta
https://deta.space/landing-page/assets/logo.20539aa2.svg
https://github.com/deta/space-docs/blob/main/.github/deta.svg

### CODE REVIEW

File: icons/detaspace/detaspace-original.svg
Comment (Snailedlt) [inline icons/detaspace/detaspace-original.svg:1]: For some reason the colors here are different than the ones inside the original-wordmark version. For example the outmost circle here is `#f73b95`, while it is `#f73b95` in the `original-wordmark` version. I'm not sure which one is the correct one, but please make sure to use the same as their official logo :)

File: devicon.json
Comment (Snailedlt) [inline devicon.json:2256]: Could you please provide a plain version, so that we can have something for the font as well?
Make sure it has a single path and a single color. Maybe it could just be the same as the original version but with some space inbetween each ring?

**Review by Snailedlt — CHANGES_REQUESTED** [review]

**Comment by Snailedlt** [comment]
@lemonyte Let us know if you need any help with this one :)

**Comment by lemonyte** [comment]
The blocker here is that a plain version of the icon would look like a plain circle. I don't want to move forward with modifications to the icon without approval from @abdelhai , so for now let's just keep this a draft.

**Comment by Snailedlt** [comment]
Fair enough!
I do have an idea for how the icon could be modified, I can try to upload an example soon so you can see if you want :)

**Comment by lemonyte** [comment]
> I can try to upload an example soon so you can see if you want 

Sure! I should have been more clear: I'm open to discuss modifications, but would like to hold off on merging any modified icons without approval 😄

**Comment by canaleal** [comment]
Small suggestion, but for the plain SVG it could be a could be a circle with white spacing in between each path. Not exactly a perfect solution, but I don't see another clear way to not make it look like a big 1 color circle. 

image

If a plain SVG cannot be made, a line SVG can be used for the font version. Similar to how the [Sketch Icon](https://github.com/devicons/devicon/blob/master/icons/sketch/sketch-line.svg) only has an original and line version. 

image

**Comment by Snailedlt** [comment]
The line version looks good!

**Comment by lemonyte** [comment]
Hi @Snailedlt, what are your thoughts on merging this? I believe all the issues have been resolved here.

**Comment by lemonyte** [comment]
It seems that the `threedsmax` and `reactrouter` icons in `devicon.json` are causing the check to fail. The `detaspace` icon is in the correct alphabetical position.

File: devicon.json
Comment (Snailedlt) [inline devicon.json:2411]: This color isn't used in any of the icons. Perhaps we could use the color of the outermost circle instead?
`#f73b95`
```suggestion
        "color": "#f73b95",
```

File: icons/detaspace/detaspace-line.svg
Comment (Snailedlt) [inline icons/detaspace/detaspace-line.svg:1]: Line and plain versions have some extra requirements. They for example need to have only 1 path. Take a look at the documentation here: 
- [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions)

You can follow this guide in order to unify the paths into one path: https://github.com/devicons/devicon/issues/1306#issuecomment-1296014473

PS: Remember to end the file with an empty line :)

File: icons/detaspace/detaspace-line-wordmark.svg
Comment (Snailedlt) [inline icons/detaspace/detaspace-line-wordmark.svg:1]: Line and plain versions have some extra requirements. They for example need to have only 1 path. Take a look at the documentation here: 
- [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)
- [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions)

You can follow this guide in order to unify the paths into one path: https://github.com/devicons/devicon/issues/1306#issuecomment-1296014473

PS: Remember to end the file with an empty line :)

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Some minor things to do before this can be merged.
PS: Don't worry aboput the sorting of devicon.json. We'll fix that in a separate PR

File: icons/detaspace/detaspace-line-wordmark.svg
Comment (lemonyte) [reply icons/detaspace/detaspace-line-wordmark.svg:1]: fixed in 09a5cc2cd62cf0e8922d30d683f062ad6fbc369e

File: icons/detaspace/detaspace-line.svg
Comment (lemonyte) [reply icons/detaspace/detaspace-line.svg:1]: fixed in 09a5cc2cd62cf0e8922d30d683f062ad6fbc369e

**Comment by lemonyte** [comment]
@Snailedlt is this good to merge? Please let me know if there is anything else to address in this PR :)

**Review by weh — APPROVED** [review]
LGTM :+1:

**Review by canaleal — APPROVED** [review]
No artifacts in the line icons:

image

**Pull request merged**
